sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -55,13 +55,11 @@ private[hive] case object NativePlaceholder extends Command
  */
 case class DescribeCommand(
     table: LogicalPlan,
-    isExtended: Boolean) extends Command {
-  override def output = Seq(
-    // Column names are based on Hive.
-    AttributeReference("col_name", StringType, nullable = false)(),
-    AttributeReference("data_type", StringType, nullable = false)(),
-    AttributeReference("comment", StringType, nullable = false)())
-}
+    override val output: Seq[Attribute] =
+      Seq(AttributeReference("col_name", StringType, nullable = false)(),
+        AttributeReference("data_type", StringType, nullable = false)(),
+        AttributeReference("comment", StringType, nullable = false)()),
+    isExtended: Boolean) extends Command
```

Comment:
I think it's OK to just change the `def` to a `val` without moving `output` into constructor argument list, so that we don't have to use named parameter `isExtended = ...` elsewhere. Did I miss something here?